### PR TITLE
Add: feature that will create new readme file for each play at the time of creating play

### PR DESCRIPTION
**Pull request by hasnainmakada-99**
# Description
Added a new file in the plop-templates which will define the heading of the readme of each play

Fixes #([47](https://github.com/atapas/react-play/issues/47))

## Type of change
Added Feature

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

`yarn create-play` by creating a sample play (Deleted it afterwards)

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/meta/play-meta.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import {
 States,
 SocialCard,
 RandomMemeGenerator,
+Sample,
```
Comment (atapas) [inline src/meta/play-meta.js:11]: This changes shouldn't becommitted

File: src/meta/play-meta.js [hunk 2]
```diff
@@ -113,6 +114,18 @@ export const plays = [
     cover: '',
     blog: '',
     video: ''
+  }, {
+    id: 'pl-sample', 
+    name: 'Sample',
```
Comment (atapas) [inline src/meta/play-meta.js:119]: shouldn't be committed

File: src/plays/index.js [hunk 3]
```diff
@@ -9,4 +9,5 @@ export { default as CounterApp } from 'plays/counter/CounterApp';
 export { default as States } from 'plays/states/States';
 export { default as SocialCard } from 'plays/social-card/SocialCard';
 export { default as RandomMemeGenerator } from 'plays/random-meme-generator/RandomMemeGenerator';
+export { default as Sample } from 'plays/sample/Sample';
```
Comment (atapas) [inline src/plays/index.js:12]: shouldn't be committed

**Review by atapas — CHANGES_REQUESTED** [review]
Only files you should be committing is play-readme.hbs and plopfile.js

**Comment by hasnainmakada-99** [comment]
Tapas sir i've made the changes which you gave me

**Review by atapas — APPROVED** [review]
Looks good.

**Pull request merged**